action.yml

```diff
@@ -6,12 +6,18 @@ branding:
   color: red
 
 inputs:
+  api_url:
+    description: 'A testing farm server url'
+    required: true
```

Comment:
default would be grade for api_url.

Reply:
You mean put there default like `https://api.dev.testing-farm.io/v0.1`?

Reply:
I guess, this one URL is correct: `https://api.testing-farm.io`

Reply:
Hmmm this one doesn't resolve.